docs/storage/prometheus.md

```diff
@@ -14,79 +14,79 @@ To monitor cAdvisor with Prometheus, simply configure one or more jobs in Promet
+`container_network_transmit_packets_total` | Counter | Cumulative count of packets transmitted | | network
+`container_network_transmit_packets_dropped_total` | Counter | Cumulative count of packets dropped while transmitting | | network
+`container_network_transmit_errors_total` | Counter | Cumulative count of errors encountered while transmitting | | network
+`container_network_tcp_usage_total` | Gauge | tcp connection usage statistic for container | | tcp
+`container_network_tcp6_usage_total` | Gauge | tcp6 connection usage statistic for container | | tcp
+`container_network_udp_usage_total` | Gauge | udp connection usage statistic for container | | udp
+`container_network_udp6_usage_total` | Gauge | udp6 connection usage statistic for container | | udp
+`container_processes` | Gauge | Number of processes running inside the container | | process
+`container_referenced_bytes` | Gauge |  Container referenced bytes during last measurements cycle based on Referenced field in /proc/smaps file, with /proc/PIDs/clear_refs set to 1 after defined number of cycles configured through `referenced_reset_interval` cAdvisor parameter.</br>Warning: this is intrusive collection because can influence kernel page reclaim policy and add latency. Refer to https://github.com/brendangregg/wss#wsspl-referenced-page-flag for more details. | bytes | referenced_memory 
 `container_spec_cpu_period` | Gauge | CPU period of the container |
 `container_spec_cpu_quota` | Gauge | CPU quota of the container |
 `container_spec_cpu_shares` | Gauge | CPU share of the container |
 `container_spec_memory_limit_bytes` | Gauge | Memory limit for the container | bytes
 `container_spec_memory_swap_limit_bytes` | Gauge | Memory swap limit for the container | bytes
 `container_spec_memory_reservation_limit_bytes` | Gauge | Memory reservation limit for the container | bytes
 `container_start_time_seconds` | Gauge | Start time of the container since unix epoch | seconds
-`container_tasks_state` | Gauge | Number of tasks in given state (`sleeping`, `running`, `stopped`, `uninterruptible`, or `ioawaiting`) |
-`container_perf_metric` | Counter | Scaled counter of perf event (event can be identified by `event` label and `cpu` indicates the core where event was measured). See [perf event configuration](docs/runtime_options.md#perf-events) |
-`container_perf_metric_scaling_ratio` | Gauge | Scaling ratio for perf event counter (event can be identified by `event` label and `cpu` indicates the core where event was measured). See [perf event configuration](docs/runtime_options.md#perf-events) |
+`container_tasks_state` | Gauge | Number of tasks in given state (`sleeping`, `running`, `stopped`, `uninterruptible`, or `ioawaiting`) | | cpuLoad
+`container_perf_metric` | Counter | Scaled counter of perf event (event can be identified by `event` label and `cpu` indicates the core where event was measured). See [perf event configuration](docs/runtime_options.md#perf-events) | | perf_event
```

Comment:
Could you investigate why perf_event is not visible in help?
```
-disable_metrics metrics
        comma-separated list of metrics to be disabled. Options are 'accelerator', 'cpu_topology','disk', 'diskIO', 'network',                                                'tcp', 'udp', 'percpu', 'sched', 'process', 'hugetlb', 'referenced_memory'. (default udp,advtcp,sched,process,hugetlb,referen                                               ced_memory,cpu_topology,tcp)
```
I use following commit 0ac6b77bee19be71f980b97c788432fc07309d76

Reply:
So, we decided that 'perf_event' is not applicable there.

Reply:
But as I understand it you need  to remove `perf_event` from the column because user cannot set for `--disable_metrics` parameter `perf_event`